gulpfile.js

```diff
@@ -88,7 +88,7 @@ gulp.task('test.e2e', ['test.compile'], function(done) {
 
   // run node with a shell so we can wildcard all the .ts files
   var cmd = 'node ../lib/main.js --translateBuiltins --basePath=. --destination=. ' +
-      '*.ts angular2/src/facade/lang.d.ts';
+      '*.ts angular2/src/facade/lang.d.ts typings/es6-promise/es6-promise.d.ts';
```

Comment:
because es6-promise is not there?
https://github.com/angular/ts2dart/blob/master/tsd.json


On Fri, Feb 12, 2016 at 4:02 PM Hans <notifications@github.com> wrote:

> In gulpfile.js
> <https://github.com/angular/ts2dart/pull/340#discussion_r52815161>:
>
> > @@ -88,7 +88,7 @@ gulp.task('test.e2e', ['test.compile'], function(done) {
> >
> >    // run node with a shell so we can wildcard all the .ts files
> >    var cmd = 'node ../lib/main.js --translateBuiltins --basePath=. --destination=. ' +
> > -      '*.ts angular2/src/facade/lang.d.ts';
> > +      '*.ts angular2/src/facade/lang.d.ts typings/es6-promise/es6-promise.d.ts';
>
> That was done a while ago though. Why is it not finding the d.ts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/angular/ts2dart/pull/340/files#r52815161>.
>